lib/counter_culture/configuration.rb

```diff
@@ -0,0 +1,36 @@
+module CounterCulture
+  def self.configuration
+    @configuration ||= Configuration.new
+  end
+
+  def self.configure
+    yield(configuration)
+  end
+
+  def self.reset_configuration
+    @configuration = Configuration.new
+  end
+
+  class Configuration
+    attr_reader :use_read_replica
+
+    def initialize
+      @use_read_replica = false
+    end
+
+    def use_read_replica=(value)
+      if value && !rails_supports_read_replica?
+        raise "Counter Culture's read replica support requires Rails 6.1 or higher"
+      end
+      @use_read_replica = value
+    end
+
+    private
+
+    def rails_supports_read_replica?
+      Rails::VERSION::STRING.to_f >= 6.1
+    rescue NameError
+      false
```

Comment:
```suggestion
      Gem::Requirement.new('>= 7.1.0').satisfied_by?(ActiveRecord.version)
```
I think it's 7.1, not 6.1 that's required? And if we use this same approach of checking the ActiveRecord version then I think that avoids needing to rescue `NameError`

Reply:
Oh yes you're right!